packages/nouns-webapp/src/components/Bid/index.tsx

```diff
@@ -247,13 +255,28 @@ const Bid: React.FC<{
             <span className={classes.customPlaceholder}>ETH</span>
           </>
         )}
-        <Button
-          className={auctionEnded ? classes.bidBtnAuctionEnded : classes.bidBtn}
-          onClick={auctionEnded ? settleAuctionHandler : placeBidHandler}
-          disabled={isDisabled}
-        >
-          {bidButtonContent.loading ? <Spinner animation="border" /> : bidButtonContent.content}
-        </Button>
+        {
+          !auctionEnded ? (
+              <Button
```

Comment:
I made the conscious choice to keep the active auction state exactly equal to status quo to make it easy to roll this back if we so choose. Happy to also remove the `auctionEnded` state code from there if people think that'd be better

Reply:
This is a good move, as it allows anyone to click the FOMO button vs the Settle button logic which only allowed connected wallets to click.